projects/media-viewer/src/lib/toolbar/main-toolbar/main-toolbar.component.html

```diff
@@ -10,14 +10,15 @@
       <mv-tb-left-pane [currentPage]="currentPage"
                        [changePageByDelta]="changePageByDeltaEvent"
                        [setCurrentPage]="setCurrentPageEvent"
-                       [highlightMode]="toolbarButtons?.highlightMode"
                        [showSidebarToggleBtn]="toolbarButtons?.showSidebarToggleBtn"
                        [showSearchbarToggleBtn]="toolbarButtons?.showSearchbarToggleBtn"
                        [showNavigationBtns]="toolbarButtons?.showNavigationBtns"
                        [searchBarHidden]="toolbarButtons?.searchBarHidden"
                        [showHighlightBtn]="toolbarButtons?.showHighlightBtn"
-                       [drawMode]="toolbarButtons?.drawMode"
-                       [sidebarOpen]="toolbarButtons?.sidebarOpen"></mv-tb-left-pane>
+                       [sidebarOpen]="toolbarButtons?.sidebarOpen"
+                       [highlightMode]="false"
+                       [drawMode]="false"
```

Comment:
how is this  going to work?

Reply:
Not sure yet, it was one of the Seams that I highlighted could be shut off without causing harm. It does need tidying up though on the next step.

Reply:
@linusnorton are you ok with this for now?

Reply:
actually removed them for the time being